alpha/apps/kaltura/modules/extwidget/actions/embedIframeJsAction.class.php

```diff
@@ -143,4 +150,44 @@ public function execute()
 		header("Location:$url");
 		KExternalErrors::dieGracefully();
 	}
+
+	/*
+	 * v2 - https://cdnapisec.kaltura.com/p/1915851/sp/191585100/embedIframeJs/uiconf_id/32880931/partner_id/1915851?iframeembed=true&playerId=kaltura_player_1719900446&entry_id=1_aeg07vpv
+	 * v7 - https://cdnapisec.kaltura.com/p/1915851/embedPlaykitJs/uiconf_id/54813242?iframeembed=true&entry_id=1_aeg07vpv
+	 */
+	private function redirectToV7($v7Id, $v2UiConfId, $partnerId, $shouldTranslatePlugins) : void
+	{
+		//validate all the params are handled
+		try
+		{
+			$config = array();
+			$config['bundleConfig'] = null;
+			$config['playerConfig'] = new stdClass();
+			v2Tov7Utils::addV2toV7config($this->getRequestParameter(v2Tov7Utils::FLASHVARS_PARAM_NAME),$v7Id);
+			if($shouldTranslatePlugins)
+			{
+				v2Tov7Utils::addV2toV7plugins(
+					$this->getRequestParameter(v2Tov7Utils::FLASHVARS_PARAM_NAME),
+					$config['bundleConfig'],
+					$config['playerConfig']);
+			}
+			//validate that the facade is reachable
+			v2Tov7Utils::getBundledFacade();
+		}
+		catch(Exception $e)
+		{
+			//if we failed, then should not redirect!
+			KalturaLog::log('V2toV7 was rejected because: ' . $e->getMessage() . ' v2 id:' .  $v2UiConfId.  ' v7 id:' . $v2UiConfId);
+			return;
+		}
+
+		$shouldTranslatePluginsQueryParam = $shouldTranslatePlugins ? '&' . v2Tov7Utils::SHOULD_TRANSLATE_PLUGINS . '=true' : '' ;
+		$host = myPartnerUtils::getCdnHost($partnerId, null , 'api');
```

Comment:
replace with internal API URL for the machines that served embedPlaykitJs Action